.github/workflows/codecoverage.yml

```diff
@@ -0,0 +1,48 @@
+name: Code Coverage
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+  push:
+    branches:
+      - 'main'
+
+env:
+  GO_VER: '1.23.0'
```

Comment:
`1.23.4` is the latest patch release. Otherwise, suggest `1.23.*` to use the latest patch release when it becomes available if we are confident it wouldn't cause unexpected issues.

Reply:
[fac8fbb](https://github.com/synadia-io/nex/pull/470/commits/fac8fbbdc995db796b097699e97abb4b90eb311f)